assets/app/scripts/services/data.js

```diff
@@ -123,21 +103,21 @@ angular.module('openshiftConsole')
 //                    by attribute (e.g. data.by('metadata.name'))
 // opts:      options (currently none, placeholder)
   DataService.prototype.list = function(resource, context, callback, opts) {
-    resource = normalizeResource(resource);
-    var callbacks = this._listCallbacks(resource, context);
+    var resourcePath = APIService.qualifyResource(resource).resource;
+    var callbacks = this._listCallbacks(resourcePath, context);
```

Comment:
qualifying the resource, then ignoring the group and version means you'll get key collisions on identical resource names in different group/versions